src/factorial.js

```diff
@@ -0,0 +1,13 @@
+export default function factorial(num){
+  if(num > 0){
+    var array = [];
+    while(num > 0){
+      array.push(num);
+        num--;}
```

Comment:
you can take this semi colon out. or you can put it on the other side of the curly brace.